src/pytorch_lightning/loggers/neptune.py

```diff
@@ -272,13 +269,10 @@ def __init__(
         agg_default_func: Optional[Callable[[Sequence[float]], float]] = None,
         **neptune_run_kwargs: Any,
     ):
+        if not _NEPTUNE_AVAILABLE:
+            raise ModuleNotFoundError(str(_NEPTUNE_AVAILABLE))
         # verify if user passed proper init arguments
         self._verify_input_arguments(api_key, project, name, run, neptune_run_kwargs)
```

Comment:
In the linked issue, the user was confused because this verification was running before the `neptune is None` check below